### PR TITLE
e2e gcs delete Test scenarios

### PR DESCRIPTION
**Pull request by itsmekumari**
@sawantpritam Please review the GCS Delete test scenarios.

### CODE REVIEW

**Comment by sawantpritam** [comment]
Always make sure there are no checkstyle errors before raising PR. @itsmekumari 
Also check indent and format.

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/locators/package-info.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Package contains the locators for the PubSub plugin.
```
Comment (sawantpritam) [inline src/e2e-test/java/io/cdap/plugin/gcsdelete/locators/package-info.java]: ???

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/runners/package-info.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Package contains the runner for the GCS sink features.
```
Comment (sawantpritam) [inline src/e2e-test/java/io/cdap/plugin/gcsdelete/runners/package-info.java]: ????

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/stepsdesign/package-info.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Package contains the stepDesign for the PubSub features.
```
Comment (sawantpritam) [inline src/e2e-test/java/io/cdap/plugin/gcsdelete/stepsdesign/package-info.java]: ??????

**Review by sawantpritam — CHANGES_REQUESTED** [review]

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/locators/package-info.java (hunk 1 above)
Comment (itsmekumari) [reply src/e2e-test/java/io/cdap/plugin/gcsdelete/locators/package-info.java]: Done.

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/stepsdesign/package-info.java (hunk 3 above)
Comment (itsmekumari) [reply src/e2e-test/java/io/cdap/plugin/gcsdelete/stepsdesign/package-info.java]: Done.

File: src/e2e-test/java/io/cdap/plugin/gcsdelete/runners/package-info.java (hunk 2 above)
Comment (itsmekumari) [reply src/e2e-test/java/io/cdap/plugin/gcsdelete/runners/package-info.java]: Done.

**Review by sawantpritam — APPROVED** [review]

**Comment by sawantpritam** [comment]
Squash the commits.

**Pull request closed without merging**